### DISCUSSION TITLE
Hangs in offset calibration

### DISCUSSION THREAD

**Discussion by kibbeling1988** [universam1/iSpindel#606] [Q&A]
I have to iSpindles and it doesn't complete the offset calibration. In Putty is see this:

         FW 7.2.1
2.2.2-dev(38a443e)
Worker run!
mounting FS... mounted!
ERROR: failed to load json config

ERROR config corrupted
scanning for OW device on pin: 5
No devices found!
scanning for OW device on pin: 12
Found device with ROM = 28 73 5E 1B 0 0 0 44
  Chip = DS18B20
  Data = 1 55 1 7F 80 7F FF B 10 5F  CRC=5F
  Temperature = 21.31 Celsius,
offsets not available
Boot-Mode: Deep-Sleep Wake

*WM: offset
Testing device connections...
MPU6050 connection successful
>************************

just a bad gyro or something else?

**Comment by ErikdBr** [comment]
> just a bad gyro or something else?

Probably, there are a lot of faulty gyros in circulation.

But what comes after 'MPU6050 connection successful' in the log? Because that is your gyro.

**Comment by wilbrod** [comment]
I'd also double check all the soldering points.
